src/util/ssa_expr.cpp

```diff
@@ -77,9 +77,10 @@ bool ssa_exprt::can_build_identifier(const exprt &expr)
 {
   if(expr.id()==ID_symbol)
     return true;
-  else if(expr.id()==ID_member ||
-          expr.id()==ID_index)
-    return can_build_identifier(expr.op0());
+  else if(expr.id() == ID_member)
+    return can_build_identifier(to_member_expr(expr).struct_op());
```

Comment:
Should be `.compound()` instead of `.struct_op()`

Reply:
ok